### PR TITLE
Add data requirements for NLDAS and GCM GLM modeling

### PR DESCRIPTION
**Pull request by hcorson-dosch-usgs**
Describe the input data requirements for modeling lakes using GLM, either with NLDAS or GCM drivers. This PR addresses part of #52

### CODE REVIEW

File: in_text/text_data_release.yml [hunk 1]
```diff
@@ -21,7 +21,14 @@ abstract: >-
   the criteria to be included in all three modeling approaches, which results in different numbers of lakes 
   in the output (noted in the file descriptions below). The input data requirements for lakes to be included 
   in the EA-LSTM predictions are lake latitude, longitude, elevation, and surface area, plus NLDAS 
-  drivers at the lake's location. All 62,966 lakes included this data release met these requirements.
+  drivers at the lake's location. All 62,966 lakes included this data release met these requirements. The input
+  data requirements for lakes to be included in the contemporary GLM NLDAS-driven predictions are lake location 
+  (within one of the following 11 states: North Dakota, South Dakota, Iowa, Michigan, Indiana, Illinois, Wisconsin, 
+  Minnesota, Missouri, Arkansas, and Ohio), latitude, longitude, maximum depth (or more detailed hypsography),
```
Comment (lindsayplatt) [inline in_text/text_data_release.yml:27]:
```suggestion
  data requirements for lakes to be included in the contemporary GLM NLDAS-driven predictions are lake latitude, longitude, maximum depth (though more detailed hypsography was used where available), 
```

**Review by lindsayplatt — APPROVED** [review]
Thanks! I don't think we need to specify the state limits because 1) it is not said anywhere in the release that we had data for more than the states we included, and 2) EA-LSTM did the same filter but we didn't list it there and should be consistent. Also, suggested slight clarification to the hypso

**Comment by hcorson-dosch-usgs** [comment]
> Thanks! I don't think we need to specify the state limits because 1) it is not said anywhere in the release that we had data for more than the states we included, and 2) EA-LSTM did the same filter but we didn't list it there and should be consistent.

The first part of that makes sense, but EA-LSTM did include lakes from more states, right? -- there are 21 unique states in the lake metadata based on the figure generated in #25. Though maybe we can cover those spatial extent differences when describing spatial coverage of the met drivers?

**Comment by lindsayplatt** [comment]
^ If the above sounds correct, then I think the text we have currently is sufficient.

**Comment by hcorson-dosch-usgs** [comment]
😬 - see [here](https://github.com/USGS-R/lake-temperature-process-models/blob/main/1_prep.R#L31) and [here](https://github.com/USGS-R/lake-temperature-process-models/blob/main/1_prep.R#L42) -- added in [this PR](https://github.com/USGS-R/lake-temperature-process-models/pull/68)

**Comment by lindsayplatt** [comment]
Le sigh. Ok, then forget my suggested edit. We should just add that filtering to states comment you made back in

**Pull request merged**